tests/from_pandas_test.py

```diff
@@ -22,6 +22,7 @@ def test_from_pandas():
 
     # Get pandas dataframe
     pandas_df = pd.DataFrame(dd_dict)
+    pandas_df['float_missing_masked'] = pandas_df['float_missing'].astype(pd.Float64Dtype())
```

Comment:
You are relying on astype to go from nan to None, I propose we add a new series with an explicit dtype, similar to int_missing.

Reply:
Done!